recipes-ni/initscripts-nilrt/initscripts-nilrt_1.0.bb

```diff
@@ -85,10 +98,16 @@ do_install () {
 	update-rc.d -r ${D} nicreatecpusets       start 1  4 5 .
 	update-rc.d -r ${D} nicreatecpuacctgroups start 2  4 5 .
 	update-rc.d -r ${D} nipopulateconfigdir   start 36 S .
-	update-rc.d -r ${D} nisetupkernelconfig   start 3  4 5 .
+	update-rc.d -r ${D} nisetupkernelconfig   start 3  5 .
```

Comment:
This script will eventually need to be on RL 4 as well, but we can do that in the later RAUC story to configure RL 4 as the new "safemode".